INSTALL.md

```diff
@@ -81,28 +83,22 @@ for more details.
 Choice of Build and Installation Procedure
 ------------------------------------------
 
-There are two partially overlapping infrastructures available to build
-Coq. They are available through `Makefile.make` (legacy / hybrid
-build) and `Makefile.dune` (full Dune build).
-
-You can use the `COQ_USE_DUNE` environment variable to change the one
-to use by default. This is useful for Coq development, where we
-recommend to rely mainly on `Makefile.dune`. Note that mixing the two
-systems is not perfectly supported and may lead to confusing behavior.
+Coq is normally built with `dune`, but a legacy/hybrid build is also
+available via `Makefile.make`.
 
 In both cases, the OCaml parts are built using
 [Dune](https://github.com/ocaml/dune). The main difference between the
 two systems is how the `.vo` files are built.
 
 In the case of `Makefile.make`, `.vo` files are built with a legacy
 Makefile, similar to what `coq_makefile` would do. In the case of
-`Makefile.dune`, `.vo` files are built with Dune, thanks to its
-recently-added, and still experimental, Coq mode.
+`dune`, `.vo` files are built with Dune, thanks to its recently-added,
```

Comment:
`Makefile.dune` had the advantage of clarifying what we meant, since `dune` is used in both places.